pytket/extensions/braket/backends/braket.py

```diff
@@ -107,6 +107,10 @@
     "name": "braket.device_schema.oqc.oqc_provider_properties",
     "version": "1",
 }
+IQM_SCHEMA = {
```

Comment:
Should update the minimum version of `amazon-braket-schemas` in `setup.py` (to `>= 1.22.0`).

Reply:
I editted the minimum version of `amazon-braket-schemas` in `setup.py`.
I couldn't try pytest for `backend_test.py` because available time of IQM on Thu and Fri is too late for me.
I'll do pytest for `backend_test.py` when IQM is online this week.

Reply:
Thanks, let me know if I can help with this -- maybe more convenient for me because of the timezone.